[blog] Polish Upcoming changes to MUI X pricing in 2024 (#43438)
mui/material-ui: docs/pages/blog/mui-x-sep-2024-price-update.md

title: Upcoming changes to MUI X pricing in 2024
description: Check out the new pricing updates and how to transition to the new model.
date: 2024-08-01T08:00:00.000Z

> authors: ['josefreitas', 'oliviertassinari']

Author: Showing my face on the blog post to take accountability for them.

authors: ['josefreitas', 'oliviertassinari']
tags: ['MUI X', 'Product']
manualCard: true
---

> We're excited to share some important updates to our product offerings and pricing structure. This change reflects the expanded feature set and helps us continue delivering high-quality components to you and your users.

Author: Wasn't sure about "excited". It's mostly about pricing and an increase, not sure exactly that users would be excited.

## Changes to the Pro plan

> ### Current pricing

Author: Sentence case.

### Current Pricing
### Current pricing

> Under our current pricing model, if you licensed the Pro plan for 50 developers, you only paid for 10 seats. This concept was introduced when we offered only the Data Grid Pro with a much smaller feature set.

Author: Removed the notion of "seat", I believe we banned it in the legal an pricing page.

Reviewer: Hmm, okay. I guess I missed that discussion.

Author: Fixing that too in #14351.

Reviewer 2: Why though? Shouldn't we have separate terms for the license (permission to use the software) and the number of devs that a single license covers?

Author: I agree but https://github.com/mui/material-ui/pull/43438#discussion_r1731566149

### New pricing

We will soon release the first Pro features of the [Charts](/x/react-charts/) and [Tree View](/x/react-tree-view/).
These new features will only be available under the updated pricing: moving forward, each developer requires a license. So, for example, 50 developers will need 50 licenses.

### Grandfathering current customers

We value our loyal customers and want to ensure a smooth transition: current customers will be grandfathered into their existing plans.

This means you can continue to renew your licenses under the legacy pricing for as long as you wish.
However, the legacy pricing only covers the previous feature scope: Data Grid and Date Pickers.

:::info
**Access to new components:** The Pro versions of Charts and Tree View, and other future Pro components, will be available exclusively through the new Pro plan model or a Premium plan (Premium customers will experience no change, all Pro components are included).
::::

### Pricing for small teams

The price is the same for customers with 10 developers or fewer.

> Actually, we are making this pricing change to be able to keep the price affordable for small teams.

Author: Not sure it's super relevant, but it's pretty much why we do this, to lower price for smaller teams, and increase for larger team who get more value.

However, if you want to use the new features, you will need a new license key to use the Pro versions of Charts or Tree View.

> You can [contact](https://support.mui.com/hc/en-us/requests/new) our support team to receive a new license key.

Author: Added a link to take action.

### Larger teams

We're happy to discuss volume discounts to accommodate larger teams.

## Changes to the Premium plan

> ### New Pro components

Author: In case people missed this.

We will soon release the first Pro features of the [Charts](/x/react-charts/) and [Tree View](/x/react-tree-view/).
This feature will be available to all Premium customers.

### Ending the early bird discount

After two years since the [introduction of our Premium plan](/blog/premium-plan-release/), we've added numerous features and are on the verge of releasing the Pivot mode for the Data Grid. We consider the product mature and battle-tested enough and as a result, the early bird discount for the Premium plan finally reaches its end.

### Large discount for current customers

However, we want to ensure that our valued early customers continue to enjoy the benefits of an earlier purchase.

> Current customers will still benefit from a 20% renewal discount (vs. 25% early bird discount before).

Author: To help appreciate it's a small increase.

title: Upcoming changes to MUI X pricing in 2024
description: Check out the new pricing updates and how to transition to the new model.
date: 2024-08-01T08:00:00.000Z
authors: ['josefreitas', 'oliviertassinari']
tags: ['MUI X', 'Product']
manualCard: true
---

Today, we are sharing some important updates to the MUI X product offerings and pricing structure.
These changes are made to reflect the expanded feature set and help support the delivering of high-quality components to you and your users.

## Changes to the Pro plan

### Current pricing

> Under the current pricing model, you only purchase a license for the first 10 developer.

Reviewer: ```suggestion
Under the current pricing model, you only purchase a license for the first 10 developers.
```

> So to license the Pro plan for 50 developers you only need to purchase 10 licenses.

Reviewer: ```suggestion
So to license the Pro plan for 50 developers you would purchase 10 licenses.
```

Btw, regarding banning "seats", I'm not sure if mixing "licenses" in plural with "license" in singular doesn't make it more confusing (and maybe even legally ambiguous)

Author: Yeah, to get a project licensed, each developer needs to be licensed. One word for two different concepts. I think we should address this a whole though.

This concept was introduced in the very beginning of the MUI X offering (2021) when we offered only the Data Grid Pro with a small feature set.

Since the release of this component, we launched several new components and features, including [Date and Time Range Pickers](/x/react-date-pickers/date-time-range-picker/).

### New pricing

We will soon release the first Pro features of the [Charts](/x/react-charts/) and [Tree View](/x/react-tree-view/).
These new features will only be available under the updated pricing: moving forward, each developer requires a license. So, for example, 50 developers will need 50 licenses.

### Grandfathering current customers

We value our loyal customers and want to ensure a smooth transition: current customers will be grandfathered into their existing plans.

This means you can continue to renew your licenses under the legacy pricing for as long as you wish.

> However, the legacy pricing only covers the previous feature scope: Data Grid and Date Pickers.

Reviewer: Rephrasing from "features scope" to "components" because I've seen users confused if there are differences in legacy Pro vs new Pro license key in Data Grid and Date pickers features as well.

```suggestion
However, the legacy pricing only covers the previously released components: Data Grid and Date Pickers.
```

:::info
**Access to new components:** The Pro versions of Charts and Tree View, and other future Pro components, will be available exclusively through the new Pro plan model or a Premium plan (Premium customers will experience no change, all Pro components are included).
::::

### Pricing for small teams

The price is the same for customers with 10 developers or fewer.
Actually, we are making this pricing change to be able to keep the price affordable for small teams.

However, if you want to use the new features, you will need a new license key to use the Pro versions of Charts or Tree View.
You can [contact](https://support.mui.com/hc/en-us/requests/new) our support team to receive a new license key.

### Larger teams

We're happy to discuss volume discounts to accommodate larger teams.

## Changes to the Premium plan

### New Pro components

We will soon release the first Pro features of the [Charts](/x/react-charts/) and [Tree View](/x/react-tree-view/).

> This feature will be available to all Premium customers.

Reviewer: ```suggestion
These features will be available to all Premium customers.
```

Reviewer: Perhaps there's an opportunity to ease customers regarding the actions they need to take.

```suggestion
These features will be available to all Premium customers without any further action needed.
```

### Ending the early bird discount

After two years since the [introduction of our Premium plan](/blog/premium-plan-release/), we've added numerous features and are on the verge of releasing the Pivot mode for the Data Grid.

> We consider the product mature and battle-tested enough and as a result, the early bird discount for the Premium plan finally reaches its end.

Author: Line break for easier comments
